frontend/src/lib/templates/frontPage/FrontPage.svelte

```diff
@@ -0,0 +1,113 @@
+<script lang="ts">
+  import {Page} from '../page';
+  import type {FrontPageProps} from './FrontPage.type';
+
+  type $$Props = FrontPageProps;
+
+  export let title: $$Props['title'];
+
+  // Merge default values to $$restProps
+  $$restProps.class = `bg-base-300 ${$$restProps.class ?? ''}`;
+  $$restProps.headerClass = `!absolute w-full bg-transparent ${$$restProps.headerClass ?? ''}`;
+  $$restProps.contentClass = `!p-0 ${$$restProps.contentClass ?? ''}`;
+</script>
+
+<!--
+@component
+The template for front pages in both the Voter and Candidate Apps.
+
+The content is provided in named slots with the default slot contents forming
+the main content.
+
+Use the properties to add to the classes of the elements containing the slots,
+to define some ids, pass Aria labels and show an optional progress bar in the
+header. You can also pass any valid properties of the `<Page>` template this
+is based on.
+  
+### Properties
+
+- `title`: The required title of the page.
+- See `FrontPage.type.ts` or code completion info.
+
+### Slots
+
+- default: main content of the page
+- `header`: content for the secondary actions displayed on the right
+  side of the `<header>`
+- `hero`: an optional hero image
+- `heading`: optional content for the `<hgroup>` block, defaults to a `<h1>` 
+  element containing the required `title` property
+- `footer`: the footer to display at the bottom of the page
+
+### Usage
+
+```tsx
+<FrontPage title="General Elections 2066">
+
+  <svelte:fragment slot="heading">
+    <p>{$page.data.appLabels.appTitle}</p>
+    <h1>{$page.data.election.name}</h1>
+  </svelte:fragment>
+
+  <img
+    slot="hero"
+    src="/images/hero.png"
+    alt="" />
+
+  <p class="text-center">
+    Lorem ipsum
+  </p>
+
+  <svelte:fragment slot="footer">
+    Footer contents
+  </svelte:fragment>
+
+</FrontPage>
+
+```
+-->
+
+<Page {title} {...$$restProps}>
+  <!-- Header -->
+  <slot name="header" slot="header" />
+
+  <!-- Default slot for Page starts -->
+
+  <!-- Hero image -->
+  {#if $$slots.hero}
+    <figure role="presentation" class="hero bg-[#d4dbef]">
+      <slot name="hero" />
+    </figure>
+  {/if}
+
+  <main class="flex flex-grow flex-col items-center justify-between pl-safelgl pr-safelgr pt-lg">
+    <!-- Title block -->
+    <hgroup class="max-w-xl py-lg text-center">
```

Comment:
This was an interesting reading. https://www.a11yproject.com/posts/how-to-accessible-heading-structure/#headings-in-html
<img width="701" alt="image" src="https://github.com/OpenVAA/voting-advice-application/assets/10437475/25e53d11-b22a-4e5a-a9ef-eda11e20a720">

Reply:
I made a tentative component to handle this behaviour, please have a look at `$lib/components/headingGroup`.

I think the other option is to consider that a specific case for the page templates and split the title into two slots in `BasicPage` so that it accepts a `mainTitle` slot and an optional `preTitle` slot, which are inserted into a similar structure.